src/logplex_message.erl

```diff
@@ -69,19 +83,17 @@ process_drains(ChannelID, Msg) ->
 process_tails(ChannelId, Msg) ->
     logplex_tail:route(ChannelId, Msg).
 
-process_redis(ChannelId, ShardInfo, Msg) ->
-    case logplex_channel:lookup_flag(no_redis, ChannelId) of
-        no_redis -> ok;
-        _ ->
-            Expiry = logplex_app:config(redis_buffer_expiry),
-            HistorySize = logplex_app:config(log_history),
-            {Map, Interval} = logplex_shard_info:map_interval(ShardInfo),
-            BufferPid = logplex_shard:lookup(integer_to_list(ChannelId),
-                                             Map, Interval),
-            Cmd = redis_helper:build_push_msg(ChannelId, HistorySize,
-                                              Msg, Expiry),
-            logplex_queue:in(BufferPid, Cmd)
-    end.
+process_redis(_ChannelId, _ShardInfo, _Msg, no_redis) ->
+    ok;
+process_redis(ChannelId, ShardInfo, Msg, _Flag) ->
```

Comment:
Unless I am misunderstanding things, not all the messages in the list are guaranteed to be for the same channel due to anychan type creds. So bumping this up to the `process_msgs/1` level is not possible.